autocomplete.js

```diff
@@ -32,18 +30,18 @@ function createAutocompleteFunction(fetchItems, { valuePath, idPath = "" }) {
 }
 
 module.exports = {
-  listBuckets: createAutocompleteFunction(
+  listBucketsAuto: createAutocompleteFunction(
```

Comment:
This function already belongs to the `autocomplete` scope, thus `auto` postfix duplicated the information about the scope. If it causes confusion at the dependent module, then it should be aliased there, not here.